src/log-timestamps.js

```diff
@@ -84,13 +111,23 @@ if (!console.constructor.hooked) {
     // jest's CustomConsole doesn't provide a debug, so we skip it as well.
   } else {
     // This path is for the normal non-jest output
-    const FUNCS = ['log', 'info', 'debug', 'error', 'warn'];
+    const FUNCS = ['info', 'debug', 'error', 'warn', 'verbose', 'silly'];
 
     for (const func of FUNCS) {
-      const realFunc = console[func];
       console[func] = function() {
-        realFunc(logPrefix() + format.apply(null, arguments));
+        const args = Array.from(arguments);
```

Comment:
This code should be replaced with `format.apply(null, arguments)` which handles objects better and is consistent with the previous code.

Reply:
Done.